src/NuGet.Server/App_Start/NuGetODataConfig.cs

```diff
@@ -28,6 +30,11 @@ public static void Initialize(HttpConfiguration config, string controllerName)
         {
             NuGetV2WebApiEnabler.UseNuGetV2WebApiFeed(config, "NuGetDefault", "nuget", controllerName);
 
+            config.Services.Replace(typeof(IExceptionLogger), new TraceExceptionLogger());
+
+            // Trace.Listeners.Add(new TextWriterTraceListener(HostingEnvironment.MapPath("~/NuGet.Server.log")));
```

Comment:
Is the commented code needed?

Reply:
This file is meant to mirror the .pp file next to it -- which is supposed to have the commented code.

Reply:
So yes 😄